hack/build/build-copy-artifacts.sh

```diff
@@ -31,12 +31,6 @@ for tgt in ${targets}; do
     if [[ "${bin_name}" == "${CLONER}" ]]; then
         mkdir -p "${CMD_OUT_DIR}/${bin_name}/"
         cp -f "${CDI_DIR}/cmd/${bin_name}/cloner_startup.sh" "${dest_dir}"
-    elif [[ "${bin_name}" == "${OPERATOR}" ]]; then
-        # This container also needs the csv-generator file, so it needs to be
-        # copied into the operator's directory for docker build to source
-        # csv-generator correctly
-        mkdir -p "${CMD_OUT_DIR}/${bin_name}/"
-        cp ${OUT_DIR}/tools/${CSV_GENERATOR}/${CSV_GENERATOR} ${OUT_DIR}/cmd/${OPERATOR}/${CSV_GENERATOR}
```

Comment:
doesn't this have to stay?

Reply:
Well only the downstream version is used, and the downstream build still has the information needed to compile and copy the binary into the downstream container. They shouldn't be using the upstream csv generator to create the HCO CSV.